README.md

```diff
@@ -37,9 +38,46 @@ Specify metrics you want to use on instantiation.
     jury = Jury(metrics=["bleu", "meteor"])
     scores = jury.evaluate(predictions, references)
 
-### CLI Usage
+### Custom Metrics
+
+You can use custom metrics with inheriting `datasets.Metric`, you can see current metrics on [datasets/metrics](https://github.com/huggingface/datasets/tree/master/metrics). The code snippet below gives a brief explanation.
+
+    import datasets
+
+    CustomMetric(datasets.Metric):
+        def _info(self):
+            pass
+        
+        def _compute(self, predictions, references, *args, **kwargs):
+            pass
+
+## <div align="center"> Contributing </div>
+
+PRs are welcomed as always :)
+
+### Installation
+
+    git clone https://github.com/obss/jury.git
+    cd pigeons
```

Comment:
Still pigeons flying around @devrimcavusoglu

Reply:
my bad, I've look for entire package now, so it should be ok.